### PR TITLE
see #543 refined explanation of git describe

### PR DESCRIPTION
**Pull request by bwl21**
HI, I tried to improve description of git describe somehow.

### CODE REVIEW

File: src/levels/mixed/describe.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿exports.level = {
```
Comment (pcottle) [inline src/levels/mixed/describe.js]: whats this change for?

**Comment by pcottle** [comment]
gonna merge but will check this works :)

**Comment by pcottle** [comment]
seems to work!

File: src/levels/mixed/describe.js (hunk 1 above)
Comment (bwl21) [reply src/levels/mixed/describe.js]: It marks this line as deleted and added. I could not find out what has been changed in this line. Maybe it is something with whitespace ...

**Pull request merged**